bin/tests/integration/config_tests.rs

```diff
@@ -289,7 +285,7 @@ define_test_config!(ipv4_only);
 define_test_config!(ipv6_only);
 define_test_config!(openssl_dnssec);
 define_test_config!(ring_dnssec);
-#[cfg(feature = "hickory-resolver")]
+#[cfg(feature = "resolver")]
```

Comment:
I think we'll need to make sure this is listed as a breaking change, right?

Reply:
oh, never mind, it was always `resolver` in the binary, this is fixing it to match as it's moved into the other crate.